sentry.gradle

```diff
@@ -161,6 +188,7 @@ gradle.projectsEvaluated {
             bundleTask.finalizedBy cliTask
           }
           previousCliTask = cliTask
+          cliTask.finalizedBy modulesTask
```

Comment:
`modulesTask` has to be executed before the packaging of the final app otherwise the modules file wont be there, I guess the packaging task should depend on `modulesTask`, I guess?